modules/xphoto/include/opencv2/xphoto.hpp

```diff
@@ -49,4 +49,5 @@
 #include "xphoto/inpainting.hpp"
 #include "xphoto/simple_color_balance.hpp"
 #include "xphoto/dct_image_denoising.hpp"
+#include "xphoto/grayworld_white_balance.hpp"
```

Comment:
can you rename the header to white_balance.hpp? since we may have other white balance algorithms and it's too much to have a header for each of them? Actually, it may be even better to rename simple_color_balance.hpp to white_balance.hpp and move your declaration there. simple_color_balance.hpp already declares grayworld algorithm, but that's fine, we will refactor it later. Your function is named differently so it there will be no conflict anyway.